### PR TITLE
conjure-up-2.2-beta3

### PR DESCRIPTION
**Pull request by adam-stokes**
This brings in improvements utilizing an asyncio library for Juju, better error
handling and logging.

Signed-off-by: Adam Stokes <battlemidget@users.noreply.github.com>

- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

**Comment by fxcoudert** [comment]
Test of the devel version fails with:

```
==> /usr/local/Cellar/conjure-up/2.2-beta3/bin/conjure-up openstack-base metal --show-env
Traceback (most recent call last):
  File "/usr/local/Cellar/conjure-up/2.2-beta3/bin/conjure-up", line 11, in <module>
    load_entry_point('conjure-up==2.2b3', 'console_scripts', 'conjure-up')()
  File "/usr/local/Cellar/conjure-up/2.2-beta3/libexec/lib/python3.6/site-packages/conjureup/app.py", line 331, in main
    show_env()
  File "/usr/local/Cellar/conjure-up/2.2-beta3/libexec/lib/python3.6/site-packages/conjureup/app.py", line 166, in show_env
    step_metas = get_step_metadata_filenames(step_scripts)
TypeError: get_step_metadata_filenames() takes 0 positional arguments but 1 was given
```

(when submitting PRs for devel versions, please also test locally with `brew install --devel` and `brew test --devel`)

File: Formula/conjure-up.rb [hunk 1]
```diff
@@ -17,7 +17,7 @@ class ConjureUp < Formula
   devel do
     url "https://github.com/conjure-up/conjure-up/archive/2.2.0-beta3.tar.gz"
     version "2.2-beta3"
-    sha256 "21349a09ec4836d6c8cf98128183fca12d1601374352974a2a32758f49ba21d1"
+    sha256 "f473d1fbbab96d5b32016f6e44c2ded94589b8286e84cd56dfbe6ff14f30bdb2"
```
Comment (fxcoudert) [inline Formula/conjure-up.rb]: The commit message says `Fix --devel test` but the commit itself is a change in SHA sum…

Comment (battlemidget) [inline Formula/conjure-up.rb]: Updated the the commit message to account for the SHA change

**Comment by battlemidget** [comment]
Beta4 is coming out this week, ill open a new PR when that lands

**Pull request closed without merging**